benchmarks/scala-dotty/src/main/scala/org/renaissance/scala/dotty/Dotty.scala

```diff
@@ -36,52 +39,37 @@ final class Dotty extends Benchmark {
 
   private val outputPath = dottyPath.resolve("output")
 
-  private val sources: mutable.Buffer[String] = mutable.Buffer[String]()
+  private var dottyBaseArgs: Seq[String] = _
 
-  private var sourcePaths: mutable.Buffer[String] = null
+  private var dottyInvocations: Seq[Array[String]] = _
+
+  private val batchCompile = false
```

Comment:
Sounds interesting to have this option exposed since you implemented both.

Could we turn this into a `@Parameter(name = "batch_compile", defaultValue =false)` ?

Reply:
Can do, but it cannot be enabled for the current set of sources -- they have name clashes and are meant to be compiled individually. This is meant to be enabled for the `scalap` sources (that are not yet in). If I remember correctly, I was getting compilation warnings and wanted to investigate them.

Reply:
Done. It did trigger a change in how benchmark access parameters, so that I can avoid changing the `BenchmarkContext` interface when I want to add new parameter type :-)